src/scripts/worldmap/MapHelper.ts

```diff
@@ -5,6 +5,7 @@ class MapHelper {
             if (this.accessToRoute(route, region)) {
                 $("[data-route='" + player.route() + "']").removeClass('currentRoute').addClass('unlockedRoute');
                 player.route(route);
+                player.region = region;
```

Comment:
Isn't player.region an observable now? So shouldn't it be `player.region(region)`

Reply:
player._region is observable, but we have get and set methods for player.region so this works fine

Reply:
Cool